dev/release/post-08-docs.sh

```diff
@@ -84,6 +84,18 @@ if [ "$is_major_release" = "yes" ] ; then
   previous_series=${previous_version%.*}
   mv docs_temp docs/${previous_series}
 fi
+# Update DOCUMENTATION_OPTIONS.theme_switcher_version_match and
+# DOCUMENTATION_OPTIONS.show_version_warning_banner
+pushd docs/${previous_version}
```

Comment:
This is `previous_series` as we move the old docs to `14.0` instead of `14.0.0`
```suggestion
pushd docs/${previous_series}
```